docs/source/pywren.scripts.rst

```diff
@@ -0,0 +1,38 @@
+pywren\.scripts package
```

Comment:
The files are auto generated. If you look at the actual docs, they look kind of weird since it just copies the docstring from the `.py` file